hw/ip/kmac/data/kmac.hjson

```diff
@@ -318,6 +318,11 @@
 
           In SwMode, the software should update the internal LFSR seed
           through !!ENTROPY_SEED_LOWER and !!ENTROPY_SEED_UPPER.
+
+          Out of the reset, the entropy module inside KMAC IP generates LFSR to
+          support the ROM_CTRL operation. To prevent let SW go back to the
```

Comment:
```suggestion
          support the ROM_CTRL operation. To prevent letting SW go back to the
```

or maybe
```suggestion
          support the ROM_CTRL operation. To prevent SW from trying to go back to the
```